www/templates/self_service_mode_login.html

```diff
@@ -0,0 +1,19 @@
+<ion-view class="company-home" cache-view="false">
+  <subheader></subheader>
+  <ion-content>
+    <div class="rc home">
+
+      <form>
+        <input numbers-only placeholder="{{('enter_pin'| translate)}}" type="number" class="input-pin show-as-pwd pin"
+               ng-model="pin.value">
+
+
+        <button ng-click="identify()"
+                class="scan-customer primary button button-block button-positive ">
+          {{ 'scanRCard' | translate }}
```

Comment:
The button should be "OK", see here: https://moqups.com/linesarefuzzy/cxlIqKWm/p:a67d89b13